Gradle on travis fails

https://travis-ci.org/akka/akka-grpc/jobs/412688002:
```
FAILURE: Build failed with an exception.
* Where:
Build file '/home/travis/build/akka/akka-grpc/gradle-plugin/build.gradle' line: 22
* What went wrong:
A problem occurred evaluating project ':akka-grpc-gradle-plugin'.
> String index out of range: -1
```
It would seem like `"git describe --tags".execute().text` is returning an empty string

--- a/.travis.yml
+++ b/.travis.yml
@@ -55,6 +55,7 @@ cache:
   directories:
     - $HOME/.ivy2/cache
     - $HOME/.sbt/boot
+    - $HOME/.gradle
 
 env:
   global:

--- a/.travis.yml
+++ b/.travis.yml
@@ -3,6 +3,10 @@ language: scala
 scala:
   - 2.12.6
 
+# make sure to fetch enough commits, so that git describe still works
+git:
+  depth: false
+
 jobs:
   include:
     - stage: tests

